searx/webapp.py

```diff
@@ -1124,22 +1123,29 @@ def image_proxy():
             except httpx.HTTPError:
                 logger.exception('HTTP error on closing')
 
+    def close_stream():
+        nonlocal resp, stream
+        try:
+            resp.close()
+            del resp
+            del stream
+        except httpx.HTTPError as e:
+            logger.debug('Exception while closing response', e)
+
     try:
         headers = dict_subset(
             resp.headers,
             {'Content-Type', 'Content-Encoding', 'Content-Length', 'Length'}
         )
-
-        def forward_chunk():
-            total_length = 0
-            for chunk in stream:
-                total_length += len(chunk)
-                if total_length > maximum_size:
-                    break
-                yield chunk
-
-        return Response(forward_chunk(), mimetype=resp.headers['Content-Type'], headers=headers)
+        response = Response(
+            stream,
```

Comment:
Important note: the size limit of 5MB is removed.

Reply:
Is it? .. in line [1080](https://github.com/searxng/searxng/pull/354/commits/a9c3c88cc0edf4dc3183f2188c518ec4a5b37055#diff-ab9553a5df12bab91ccac7a4b9af78fbd160e35bffae9a340bf75ed736d50a07R1080) there is a `maximum_size` which is checked in line 1100  `if ... and int(content_length) > maximum_size`